src/react/src/components/FacilityDetailsItem.jsx

```diff
@@ -7,7 +7,7 @@ import FacilityDetailsDetail from './FacilityDetailsDetail';
 import TitledDrawer from './TitledDrawer';
 import ShowOnly from './ShowOnly';
 
-const detailssStyles = theme =>
+const detailsStyles = theme =>
```

Comment:
I just fixed a typo that I noticed. This change doens't directly related to the ProductionLocationDialog component.